src/MiniCover/Instrumentation/Instrumenter.cs

```diff
@@ -333,15 +333,17 @@ private void CreateAssemblyInit(AssemblyDefinition assemblyDefinition)
 
         private string GetSourceRelativePath(string path)
         {
-            if (!path.StartsWith(normalizedWorkDir))
-                return null;
-
-            if (!sourceFiles.Contains(path))
-                return null;
-
-            return path.Substring(normalizedWorkDir.Length);
+            Uri file = new Uri(path);
+            Uri folder = new Uri(normalizedWorkDir);
+            string relativePath = 
+                Uri.UnescapeDataString(
+                    folder.MakeRelativeUri(file)
+                        .ToString()
+                        .Replace('/', Path.DirectorySeparatorChar)
+                );
+            return  relativePath;
```

Comment:
nitpick, but there seems to be an extra space between `return` and `relativePath`